docs/gettingstarted/podman-systemd-general.md

```diff
@@ -82,26 +80,25 @@ sudo systemctl stop sc4s
 # Verify Proper Operation
 
 SC4S has a number of "preflight" checks to ensure that the container starts properly and that the syntax of the underlying syslog-ng
-configuration is correct.  After this step completes, to verify SC4S is properly communicating with Splunk,
-execute the following search in Splunk:
+configuration is correct.  After this step is complete, verify SC4S is properly communicating with Splunk by
+executing the following search in Splunk:
 
 ```ini
 index=* sourcetype=sc4s:events "starting up"
 ```
 
-This should yield an event similar to the following:
+This should yield an event similar to the following when the startup process proceeds normally (without syntax errors). 
```

Comment:
can we cut off `(without syntax errors)`? I know it was in the doc before but other kinds of errors can disrupt the startup process as well

Reply:
`(without syntax errors)` remained